packages/blocks/src/api/parser.js

```diff
@@ -115,6 +115,11 @@ export function matcherFromSource( sourceConfig ) {
 		case 'query':
 			const subMatchers = mapValues( sourceConfig.query, matcherFromSource );
 			return query( sourceConfig.selector, subMatchers );
+		case 'tag':
```

Comment:
Can we add unit tests for this new attribute source?

Reply:
There's no tests for the other sources either. Could add tests for all of them though... Not sure if valuable.

Reply:
My gut says we should have them since this is an API we expose to third party block authors. But if there's none in `master` then we can do this separately! 👌